include/map.h

```diff
@@ -31,7 +31,7 @@ typedef struct MapConfig {
     } tattle;
 } MapConfig; // size = 0x40
 
-typedef s32 MapInit(void);
+typedef s32(*MapInit)(void);
```

Comment:
Is it possible to have a space after `s32` here? Or does format.sh get rid of it?

Reply:
I think I initially did have it. I'll add it back and then format just this and see if it goes away